app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] OPTIONAL
There are commented blocks found in your code on the repository. Kindly endeavor to remove **all** commented blocks from your code before pushing it to the repository so as to keep your code cleaner.